flocker/provision/_ssh/_conch.py

```diff
@@ -101,7 +101,7 @@ def perform_put(dispatcher, intent):
     """
     See :py:class:`Put`.
     """
-    return Effect(Run(command='printf %s > %s'
+    return Effect(Run(command='printf -- %s > %s'
                               % (shell_quote(intent.content),
                                  shell_quote(intent.path))))
 
```

Comment:
Provisioning fails if you don't make that change, since certificates start with `--`.